v8/client/client.go

```diff
@@ -145,7 +145,7 @@ func (cl *Client) IsConfigured() (bool, error) {
 	}
 	// Client needs to have either a password, keytab or a session already (later when loading from CCache)
 	if !cl.Credentials.HasPassword() && !cl.Credentials.HasKeytab() {
-		authTime, _, _, _, err := cl.sessionTimes(cl.Credentials.Domain())
+		authTime, _, _, _, _, err := cl.sessionTimes(cl.Credentials.Domain())
```

Comment:
The dogsledding is becoming excessive here, I'm not entirely sure if this isn't worth a refactor. However, that's not our problem, it should be fixed upstream.

Reply:
Indeed it is, though it is not our problem currently. If we end up properly forking the library we can rebase it later.